pkg/scheduler/apis/config/v1alpha1/defaults.go

```diff
@@ -62,7 +62,7 @@ func SetDefaults_KubeSchedulerConfiguration(obj *kubescedulerconfigv1alpha1.Kube
 		}
 		obj.HealthzBindAddress = net.JoinHostPort(host, port)
 	} else {
-		obj.HealthzBindAddress = net.JoinHostPort("0.0.0.0", strconv.Itoa(ports.SchedulerPort))
+		obj.HealthzBindAddress = net.JoinHostPort("0.0.0.0", strconv.Itoa(ports.InsecureSchedulerPort))
```

Comment:
how is the secure port controlled? I expected parallel config fields to allow it to be specified

Reply:
we have not changed the component config. We want to unify the serving config anyway. So these secure ports give us the chance. This will be a follow-up. For now it is flag only.